ks_includes/config.py

```diff
@@ -229,7 +229,8 @@ def _create_configurable_options(self, screen):
                 "range": [50, 150], "step": 1}},
             {"autoclose_popups": {"section": "main", "name": _("Auto-close notifications"), "type": "binary",
                                   "value": "True"}},
-
+            {"show_heater_power": {"section": "main", "name": _("Show Heater Power"), "type": "binary",
+                                   "value": "False"}},
```

Comment:
nice!

Reply:
Only thing that could be better still is that when that setting is changed KlipperScreen has to restart for it to become effective.